package.json

```diff
@@ -16,7 +16,7 @@
     "url": "https://github.com/Microsoft/react-native-code-push"
   },
   "dependencies": {
-    "code-push": "^1.1.1-beta",
+    "code-push": "1.5.0-beta",
```

Comment:
We'll need to update this to `1.5.1-beta` to accommodate the `package` fix I have a PR out for.